docs: remove CV postcondition examples (#380)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/health.mdx

## Continuous Validation

> You can use [`check`](/terraform/language/checks) blocks to create custom rules to validate your infrastructure's resources, data sources, and outputs.

Reviewer: ```suggestion
Continuous validation will evaluate both preconditions and postconditions and check blocks as part of an assessment, but we recommend using [`check` blocks](/terraform/language/checks) for post-apply monitoring. Use `check` blocks to create custom rules to validate your infrastructure's resources, data sources, and outputs.
```

The page shows all of the resources, outputs, and data sources with custom assertions that Terraform Cloud evaluated. Next to each object, Terraform Cloud reports whether the assertion passed or failed. If one or more assertions fail, Terraform Cloud displays the error messages for each assertion.

The health assessment page displays each assertion by its [named value](/terraform/language/expressions/references). A `check` block's named value combines the prefix `check` with its configuration name.

> #### Preconditions and Postconditions

Reviewer: ```suggestion
```

> It is recommended that `check` blocks are used for post-apply monitoring. However, any [Preconditions and postconditions](/terraform/language/expressions/custom-conditions#preconditions-and-postconditions) defined in your configuration will also be evaluated by Continuous Validation. The results of these assertions will be displayed alongside the results of any assertions from `check` blocks, identified by the named values of their parent block.

Reviewer: ```suggestion
```
